<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>150</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WinPEFix</string>
  </property>
  <property name="windowIcon">
   <iconset theme="application-x-desktop">
    <normaloff/>
   </iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QPushButton" name="selectPushButton">
      <property name="text">
       <string>Select Files...</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" rowspan="3">
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="lineWrapMode">
       <enum>QPlainTextEdit::NoWrap</enum>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="processPushButton">
      <property name="text">
       <string>Process</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
